src/wallet/wallet2.cpp

```diff
@@ -1924,7 +1793,6 @@ void wallet2::refresh(uint64_t start_height, uint64_t & blocks_fetched, bool& re
       waiter.wait();
       if(blocks_start_height == next_blocks_start_height)
       {
-        m_node_rpc_proxy.set_height(m_blockchain.size());
```

Comment:
Why remove this (the RPC class also caches this) ?

Reply:
The functionality that the node proxy class was serving is moved to the rpc client, yes.

Reply:
I meant, why is it removed, as opposed to doing this on your new RPC class ? Not doing it means a possible lag in caching, which is probably not too bad, but still unnecessary.